docs/news/aragon.md

```diff
@@ -1,28 +1,35 @@
+|[Read More](https://blog.aragon.one/aragon-nest-first-awarded-grants-29a2f3aa8d4a)|[Read More](https://blog.aragon.one/aragon-q1-development-update-b5525e9477c4)|[Read More](https://blog.aragon.one/first-community-wide-aragon-feedback-survey-132df9547fbc)|
 
-| | |  
-:-----------:|:-----------:|:-----------:|  
-| | |  
-| | |  
-| | |  
-| | |  
-| | |  
+|[Introducing Aragon One](https://blog.aragon.one/introducing-aragon-one-b14dd804c5ce)|[Decentralizing Aragon’s development III: Onboarding new teams](https://blog.aragon.one/decentralizing-aragons-development-iii-onboarding-new-teams-32786cb805a5)| [Aragon Releases New Developer Portal For Building DApps and DAOs](https://sludgefeed.com/aragon-releases-new-developer-portal-for-building-dapps-and-daos/)|
+:-----------:|:-----------:|:-----------:
+|[Aragon](aragon.md)|[Aragon](aragon.md)|[Aragon](aragon.md)
+|[<img src=https://cdn-images-1.medium.com/max/2000/1*PPdfJNy3uNaegc7QYqTk2w.png>](https://blog.aragon.one/introducing-aragon-one-b14dd804c5ce)|[<img src=https://cdn-images-1.medium.com/max/1000/1*FlHb4MYFi9eyxPOvwHTm8Q.png>](https://blog.aragon.one/decentralizing-aragons-development-iii-onboarding-new-teams-32786cb805a5)|[<img src=https://sludgefeed.com/wp-content/uploads/2018/05/Aragon-696x391.png>](https://sludgefeed.com/aragon-releases-new-developer-portal-for-building-dapps-and-daos/)
+|Author [Luis Cuende](https://twitter.com/licuende) on May 17|Author [Luis Cuende](https://twitter.com/licuende) on May 18|Author [Craig Russo](https://twitter.com/craig_a_russo) on May 21|
+|Since we released the first version of the Aragon whitepaper, we knew that it was important to split the non-profit entity overseeing the project from the actual companies or teams working on its development. Today, we are announcing the next step. We are announcing Aragon One.|We have already explained why we are decentralizing Aragon’s development, how we are decentralizing the legal vehicle that represents the project and introduced Aragon One, the first company contributing to Aragon’s development. In this post, we will lay out the strategy for onboarding more teams to work on core Aragon infrastructure.|Aragon (ANT), a project facilitating the creation and maintenance of organizational structures by using blockchain technology, announced the launch of a new developer portal today, providing developers with a new command line tool and improved documentation for building decentralized Aragon apps (DApps).|
+|[Read More](https://blog.aragon.one/introducing-aragon-one-b14dd804c5ce)|[Read More](https://blog.aragon.one/decentralizing-aragons-development-iii-onboarding-new-teams-32786cb805a5)|[Read More](https://sludgefeed.com/aragon-releases-new-developer-portal-for-building-dapps-and-daos/)|
+
+|[Releasing the Aragon Developer Portal](https://blog.aragon.one/releasing-the-aragon-developer-portal-237a8dbc8a48)|[Aragon Q1 Transparency Report](https://blog.aragon.one/aragon-q1-transparency-report-5ba9fc435ca6)|
```

Comment:
Table is missing the third, empty, column.

Reply:
Done